<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigCredentialHelperDialog</class>
 <widget class="QDialog" name="ConfigCredentialHelperDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QRadioButton" name="radioButton_wincred">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>86</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>wincred</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_winstore">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>86</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>winstore</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_none">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>86</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>None</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_other">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>86</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Other</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_helper">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>150</y>
     <width>351</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ConfigCredentialHelperDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>ConfigCredentialHelperDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>317</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
